ginivision/src/main/java/net/gini/android/vision/digitalinvoice/DigitalInvoiceActivity.kt

```diff
@@ -63,6 +95,20 @@ class DigitalInvoiceActivity : AppCompatActivity(), DigitalInvoiceFragmentListen
                 EDIT_LINE_ITEM_REQUEST)
     }
 
+    override fun onPayInvoice(selectedLineItems: List<LineItem>, selectedLineItemsTotalPrice: String, deselectedLineItems: List<LineItem>,
+                              reviewedCompoundExtractions: Map<String, GiniVisionCompoundExtraction>,
+                              reviewedExtractions: Map<String, GiniVisionSpecificExtraction>) {
+        setResult(Activity.RESULT_OK, Intent().apply {
+            putExtra(CameraActivity.EXTRA_OUT_EXTRACTIONS, Bundle().apply {
+                reviewedExtractions.forEach { putParcelable(it.key, it.value) }
+            })
+            putExtra(CameraActivity.EXTRA_OUT_COMPOUND_EXTRACTIONS, Bundle().apply {
```

Comment:
Why are those defined on the CameraActivity?

Reply:
Because the CameraActivity is the entry point when using the Screen API. It is started for result and then from the client's point of view it's expected that the keys for the returned extras are part of the CameraActivity.

Reply:
👍🏼